nucypher/cli/commands/status.py

```diff
@@ -116,13 +117,23 @@ def network(general_config, registry_options):
 @status.command()
 @group_registry_options
 @option_staking_address
+@click.option('--substakes', help="Print all sub-stakes for this staker", is_flag=True, default=False)
 @group_general_config
-def stakers(general_config, registry_options, staking_address):
+def stakers(general_config, registry_options, staking_address, substakes):
     """Show relevant information about stakers."""
+    if substakes and not staking_address:
+        raise click.BadOptionUsage(option_name="--substakes",
+                                   message="--substakes is only valid when used with --staking-address.")
     emitter, registry, blockchain = registry_options.setup(general_config=general_config)
     staking_agent = ContractAgency.get_agent(StakingEscrowAgent, registry=registry)
     stakers_list = [staking_address] if staking_address else staking_agent.get_stakers()
     paint_stakers(emitter=emitter, stakers=stakers_list, registry=registry)
+    staker = Staker(registry=registry,
+                    domain=registry_options.network,
+                    checksum_address=staking_address)
+    if substakes:
+        staker.stakes.refresh()
```

Comment:
Should we flip this?

```suggestion
    if substakes:
        staker = Staker(registry=registry,
                        domain=registry_options.network,
                        checksum_address=staking_address)
        
        staker.stakes.refresh()
```

Reply:
✔️ - Good eye - this was causing a bug.